diff --git a/src/dxvk/dxvk_adapter.cpp b/src/dxvk/dxvk_adapter.cpp
index 24c7cea7..c099d719 100644
--- a/src/dxvk/dxvk_adapter.cpp
+++ b/src/dxvk/dxvk_adapter.cpp
@@ -311,7 +311,7 @@ namespace dxvk {
       m_deviceFeatures.khrBufferDeviceAddress.bufferDeviceAddress;
 
     if (enableCudaInterop) {
-      devExtensions.nvxBinaryImport.setMode(DxvkExtMode::Optional);
+      // devExtensions.nvxBinaryImport.setMode(DxvkExtMode::Optional);
       devExtensions.nvxImageViewHandle.setMode(DxvkExtMode::Optional);
       devExtensions.khrBufferDeviceAddress.setMode(DxvkExtMode::Optional);
